26-register_registration/resource_types/artistic_result/creator_selects_resource_type_artistic_result_and_subtype_literary_arts.feature

```diff
@@ -0,0 +1,70 @@
+Feature: Creator selects Resource type Artistic Result and subtype Literary Arts
+
+  Scenario: Creator navigates to the Resource Type tab and selects Resource subtype "Literary Arts"
+    Given Creator navigates to the Resource Type tab and selects Resource type "Artistic Result"
+    When they select Resource Subtype "Literary Arts"
+    Then they see fields:
+      | More information |
+    And they see field for Type Work with options:
+      | Novel        |
+      | Poetry       |
+      | Novella      |
+      | ShortFiction |
+      | Essay        |
+      | Translation  |
+      | Retelling    |
+      | Play         |
+      | Other        |
+    And they can add Announcements of type:
+      | Monograph                |
+      | Web Publication          |
+      | Performance              |
+      | Audio/Visual Publication |
+    And they see that each Announcement has a Remove Button
+    And they see that each Announcement has an Edit Button
+    And they see an Add Announcement Button
+
+  Scenario: Creator adds an Monograph
+    Given Creator navigates to the Resource Type tab and selects Resource subtype "Literary Arts"
+    When they add a Monograph with details for:
+      | Publisher   |
+      | Year        |
+      | ISBN        |
+      | Total pages |
+    Then the Monograph is listed under Announcements
+
+  Scenario: Creator adds an Web Publication
+    Given Creator navigates to the Resource Type tab and selects Resource subtype "Literary Arts"
+    When they add a Web Publication with details for:
+      | Link      |
+      | Publisher |
+      | Year      |
+    Then the Web Publication is listed under Announcements
+
+  Scenario: Creator adds an Performance
+    Given Creator navigates to the Resource Type tab and selects Resource subtype "Literary Arts"
+    When they add a Performance with details for:
+      | Type of Performance |
+      | Place               |
+      | Date                |
+    And Type of Performance can be one of:
+      | Reading |
+      | Play    |
+      | Other   |
+    Then the Performance is listed under Announcements
+
+  Scenario: Creator adds an Audio/Visual Publication
+    Given Creator navigates to the Resource Type tab and selects Resource subtype "Literary Arts"
+    When they add an Audio/Visual Publication with details for:
+      | Type of audio/visual publication |
+      | Publisher                        |
+      | Year                             |
+      | ISBN                             |
+      | Duration                         |
+    And Type of Type of audio/visual publication can be one of:
+      | Audiobook |
+      | RadioPlay |
+      | ShortFilm |
+      | Podcast   |
+      | Other     |
+    Then the Audio/Visual Publication is listed under Announcements
```

Comment:
Legg til scenario for Delete og Edit Announcement

Reply:
Holder kanskje med "they can edit/delete", så slipper vi å låse oss opp mot implementasjonsdetaljer?